zetaclient/zetacore/tx.go

```diff
@@ -69,6 +70,7 @@ func GasPriceMultiplier(chainID int64) (float64, error) {
 	return 0, fmt.Errorf("cannot get gas price multiplier for unknown chain %d", chainID)
 }
 
+// WrapMessageWithAuthz wraps a message with an authz message
```

Comment:
We can mention why we do this, as in hotkey address is the one actually broadcasting the transactions, instead of the operator